src/lib/Scenes/Home/Components/ForYou/Components/ArtworkCarouselHeader.tsx

```diff
@@ -172,9 +177,13 @@ interface RelayProps {
   rail: {
     title: string | null
     key: string | null
-    context: Array<boolean | number | string | null> | null
+    context: any
   }
 }
+
+// FIXME: What is this workaround? Can we stop using it?
```

Comment:
It’s probably a workaround in the sense that `relay2ts` didn’t emit the right typings and so this interface was added to override the faulty prop. This will all go away soon.